Sources/SKSupport/FileSystem.swift

```diff
@@ -16,22 +16,24 @@ import struct TSCBasic.AbsolutePath
 
 /// The home directory of the current user (same as returned by Foundation's `NSHomeDirectory` method).
 public var homeDirectoryForCurrentUser: AbsolutePath {
-  return AbsolutePath(NSHomeDirectory())
+  get throws {
+    return try AbsolutePath(validating: NSHomeDirectory())
```

Comment:
Just a thought: We are throwing if `NSHomeDirectory` doesn’t validate but using `try!` for `NSTemporaryDirectory` below. Would it be fair to assume that `NSHomeDirectory` always passes validation and we can do `try!` here?